### DISCUSSION TITLE
zh-cn/manual/device/macos

### DISCUSSION THREAD

**Discussion by giscus** [MaaAssistantArknights/maa-website#142] [Comments]
# zh-cn/manual/device/macos

Mac 模拟器 Apple Silicon 芯片 ✅ PlayCover（原生运行最流畅 🚀） 试验性支持，遇到问题请多多提 issue，并在标题中提及 iOS。 注意：由于 macOS 本身机制的问题，将游戏窗口最小化、台前调度状态下切换到别的窗口、将窗口移动到别的桌面/屏幕之后，截图会出现问题，导致无法正确运行。相关 Issue #4371 要求...

https://docs.maa.plus/zh-cn/manual/device/macos.html

**Comment by TPOFGG** [comment]
请问直接在Mac的AppStore里安装的游戏客户端可以连接使用maa吗👀

**Reply by Rbqwow** [reply]
maa只能通过adb连接和控制 所以不能
至少现在不能

**Reply by TPOFGG** [reply]
请问按照PlayCover那个类型的所有步骤做完了以后，连接的时候不能成功是为什么呢（一点开始马上就自动停止了）

**Reply by Daydreamer114** [reply]
参照
https://github.com/MaaAssistantArknights/MaaAssistantArknights/issues/10192
进行尝试

**Comment by raoxiaoyang** [comment]
想问下有没有b服的安装包

**Reply by Toby-Shi-cloud** [reply]
appstore似乎没有粥的B服。用安卓模拟器就行了吧。

**Reply by liuyuxiang736-droid** [reply]
B服api没有的

**Comment by dwtnan** [comment]
Mac的MAA显示“更新错误：在下载过程中出现错误”

**Reply by dwtnan** [reply]
我还是重新下载一下吧

**Reply by snail-blip** [reply]
我也是这个问题，而且也有mirror酱的cdk还是这个问题，

**Comment by JoyeLiang** [comment]
请问在PlayCover中“绕过“MaaTools”，运行明日方舟也不显示端口号应该怎么办呢？

**Comment by scarlet1661** [comment]
如果我用ios M系列芯片的Mac的话MAA该如何连接AVD。文档里面似乎对这方面描写不多

**Reply by ABA2396** [reply]
用 adb 连

**Reply by scarlet1661** [reply]
请问AVD的adb在哪里啊？好像找不到

**Comment by ZhaoqingT** [comment]
我的PlayCover里面没有maaTools选项，playcover 3.1.0 我应该在playcover里面问还是这里问？

**Reply by FitzSnow** [reply]
看文档第一步

**Comment by CharlesLu1207** [comment]
请问我都按照步骤做了，但是一直显示正在连接，可能是什么问题（Apple Silicon）

**Reply by CharlesLu1207** [reply]
显示正在运行中但是没有任何操作

**Reply by LagrangeMiddleFinger** [reply]
俺也一样

**Reply by ABA2396** [reply]
cc @hguandl

**Comment by ruibuaichiyu** [comment]
mac但是是windows 芯片安装哪个😭

**Comment by MoxyParis** [comment]
Apple Silicon的play cover无法连接，显示如下：

正在连接……
如果长时间连接不上或出错，请尝试下载使用“文件”》“PlayCover链接…”中的最新版本
正在运行中……
连接失败
请「检查连接设置」或「尝试重启模拟器与ADB」或「重启电脑」

**Comment by snail-blip** [comment]
更新之后，过两秒直接显示  更新...   看其他人的反馈应该是更新失败，原因未知，唯一的更新方式只能是回官网重新下载，这里是issue吗？

**Comment by cloud-oc** [comment]
现在AppStore上下载的客户端能连Maa吗

**Reply by youzic37** [reply]
现在可以了吗

**Reply by cloud-oc** [reply]
不知道，没试过，还是用的playcover

**Comment by toa-heavyindustries** [comment]
Bluestacks Air模拟器，EW三技能开了之后秒关，有办法解决吗

**Comment by Ezhixuan** [comment]
macOS26 中 playcover 按键映射存在问题没办法关闭,有意愿解决这个问题吗,或者有没有哪个哥们设置好了导出一份按键给我求求

**Comment by liuyuxiang736-droid** [comment]
使用playcover标题栏结尾没有出现端口号

**Reply by liuyuxiang736-droid** [reply]
哦不对下载错的版本了

**Comment by longisphotoing-max** [comment]
标题栏端口号设置都没有问题，但还是现实连接失败☹️之前挂VPN可以用，今天挂了也无法使用

**Comment by GreyCat0122** [comment]
反馈一个问题，MAC 版本的 MAA 在勾选“使用 ABD-Lite 连接”后会出现疑似休息室点击速度过快导致不放置疲劳干员或者放置少量疲劳干员的情况，多次复现后发现不勾选此项确实没有这个BUG 了，设备为 M4的MBA ，不影响使用，其余一切正常，感谢开发组

**Comment by ET-yzk** [comment]
想请问一下m2的mac，playcover，使用了开始唤醒和自动关闭功能，但是会存在在启动明日方舟时遇到明日方舟奔溃的问题，mac会有一个重新启动的提示弹窗（重新打开几次可以正常启动），导致maa一直卡着（我体感手动关闭遇到这个问题的概率比使用maa自动关闭要小），有人遇到过相似的问题嘛？

**Comment by AOO325799** [comment]
本人用的Intel Mac(2.6 GHz 六核Intel Core i7),目前蓝叠网站貌似看不到适配我这个的版本(全不显示),夜神因为打开明日方舟紫屏不行(大概显卡问题),AVD没试过,不过好在还有磁盘转换win,但我还是想在mac里玩到明日方舟啊QwQ

**Comment by snail-blip** [comment]
已收到

**Reply by ABA2396** [reply]
再收到就ban账号了

**Comment by guu120** [comment]
- 下载了https://github.com/PlayCover/PlayCover版本的PlayCover 3.1.0
- 游戏脱壳安装包也可以正常运行
！- 在 PlayCover 中右键app，选择 设置 - 绕过，勾选 启用 PlayChain、启用绕过越狱检测、插入内省库，这里没有MaaTools，然后标题栏结尾也没有端口号，maa连接时提示无法连接。是PlayCover 3.1.0版本的问题吗？文档链接里的PlayCover安装完后提示损坏无法运行

**Reply by guu120** [reply]
m1芯片的mac

**Reply by guu120** [reply]
已经解决了，感谢作者

**Comment by Nydia-Adler** [comment]
打开maa 显示 初始化失败: 未能完成操作。（MAA.MaaCoreError错误0。）

**Reply by ABA2396** [reply]
下 v6.17.0-beta.6

**Comment by snail-blip** [comment]
已收到
